src/lib/apiClient.js

```diff
@@ -40,6 +40,14 @@ const getCollections = (bastionName, callback) => {
     .catch(e => console.log(e))
 };
 
+const getCollection = (bastionName, collectionName, callback) => {
```

Comment:
Added this. We fetch the records of the `active` collection. This might not be ideal actually, but my objective was to make something work to test styling